README.md

```diff
@@ -16,45 +28,75 @@ video:
 
 - Dynamic Instrumentation (No Code Change Required)
 - Robust Sampling
```

Comment:
Again, I know it was already there, but "Robust"? What about "Callstack Sampling".

Reply:
Agree.